cmd/key_generate.go

```diff
@@ -12,26 +12,25 @@ import (
 )
 
 var (
-	genKeyArgs struct {
+	keyGenerateArgs struct {
 		name       string
 		passphrase string
 		metas      string
 	}
 
-	// genKeyCmd represents the genkey command
-	genKeyCmd = &cobra.Command{
-		Use:   "genkey",
+	keyGenerateCmd = &cobra.Command{
+		Use:   "generate",
 		Short: "Generate a new key pair for a wallet",
 		Long:  "Generate a new key pair for a wallet, this will implicitly generate a new wallet if none exist for the given name",
 		RunE:  runGenKey,
 	}
 )
 
 func init() {
-	rootCmd.AddCommand(genKeyCmd)
-	genKeyCmd.Flags().StringVarP(&genKeyArgs.name, "name", "n", "", "Name of the wallet to use")
-	genKeyCmd.Flags().StringVarP(&genKeyArgs.passphrase, "passphrase", "p", "", "Passphrase to access the wallet")
-	genKeyCmd.Flags().StringVarP(&genKeyArgs.metas, "metas", "m", "", `A list of metadata e.g: "primary:true;asset:BTC"`)
+	keyCmd.AddCommand(keyGenerateCmd)
+	keyGenerateCmd.Flags().StringVarP(&keyGenerateArgs.name, "name", "n", "", "Name of the wallet to use")
+	keyGenerateCmd.Flags().StringVarP(&keyGenerateArgs.passphrase, "passphrase", "p", "", "Passphrase to access the wallet")
+	keyGenerateCmd.Flags().StringVarP(&keyGenerateArgs.metas, "meta", "m", "", `A list of metadata e.g: "primary:true;asset:BTC"`)
 }
 
 func runGenKey(cmd *cobra.Command, args []string) error {
```

Comment:
Do you want to rename this function keyGenerate or something maybe? (don't really mind, just as you changed thhe names...)